.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,3 +29,6 @@ Fixes #(issue)
 - [ ] I ran `npm run build-all` and tested the resulting files from `dist` folder in a browser.
 - [ ] I have updated the `CHANGELOG.md` file in the root folder.
 - [ ] I have tested my code on the live network.
+- [ ] The browser visual inspection check looks ok: [sudden-playground.surge.sh][1]
```

Comment:
cool, but might be a little confusing since other PRs can overwrite it at any time. just thinking out loud, i wonder if we could "append" to the `index.html` in surge, that would be cool to have a running log of successful (or not) results.

Reply:
Yes this would be ideal: e.g `sudden.playground.surge.sh/<branch>/<commit>`

As I read [surge issue 119][1], there's no built-in support for this per se and we'd need to maintain our own store of previously published pages. 

We can definitely highlight the potential overwrite more clearly in the published page though, will amend this PR for that.
 
[1]: https://github.com/sintaxi/surge/issues/119

Reply:
834cb78